### DISCUSSION TITLE
Launcher not working with 512m heap size

### DISCUSSION THREAD

**Discussion by rutledj** [nextgenhealthcare/connect#5843] [Q&A]
For some reason, when I try to start one of our mirth instances (we have 8), I get a heap memory error. Checking the appliance it shows it is only using 45% of available memory. My pc also has plenty of memory available.  If I change the heap size to 256 it will start. 

Guess I'm not really clear on what this setting it doing or if it is related to the Mirth memory allocation on the appliance or running on my PC.  This is the first time I've seen this error when launching MC.

**Comment by jonbartels** [comment]
What specific errors do you get?

The memory setting in the launcher is how much memory the MC client will use when it launches. Larger MC instances (500+ channels) often need ALL THE RAM to function. Most instances are fine on 256-512 MB. This setting is completely disassociated from the settings used by the server to actually run the Mirth server processes.

If you do not get any errors, start the launcher by:

1. Open the command line
1. `cd` to its installation directory
2. `cd java/app`
3. `java -jar mirth-client-launcher.jar`
4. You'll see the Launcher GUI. Launch Mirth so that you reproduce the error
5. The command line will show output. Copy and paste that output here **USING CODE FORMATTING SO WE CAN READ IT**

**Comment by rutledj** [comment]
Well, for some reason it is no longer giving that error. Working fine with 512 selected. I'll keep these instructions in case it comes back.
